nengo_gui/namefinder.py

```diff
@@ -22,9 +22,12 @@ def find_names(self, net):
         classes = (nengo.Node, nengo.Ensemble, nengo.Network,
                    nengo.Connection)
 
+        private = False
+        in_lists = False
```

Comment:
Because I forgot that it doesn't matter in Python if you initialise variables inside or outside a for-loop. http://stackoverflow.com/questions/407255/difference-between-declaring-variables-before-or-in-loop